handlers/main.yml

```diff
@@ -5,9 +5,25 @@
     daemon_reload: true
     name: prometheus
     state: restarted
+  when:
+    - not prometheus_docker_install
+
+- name: restart container
+  become: true
```

Comment:
This shouldn't be executed with `become: true` as it overwriting all security measures of target system